services/identity/service.go

```diff
@@ -39,14 +43,14 @@ func init() {
 // Service handles identities
 type Service struct {
 	*sda.ServiceProcessor
-	identities      map[string]*storage
+	Identities      map[string]*Storage
 	identitiesMutex sync.Mutex
```

Comment:
Well I use `Service` as the struct you're describing. So you would prefer having yet another struct and put that in here?